### DISCUSSION TITLE
Not having a university email

### DISCUSSION THREAD

**Discussion by crunchiness01** [community/community#131882] [GitHub Education]
### Hi everyone,

Hello everyone,
My university does not provide university-email to undergraduate students, 
And my request for "Github education" is rejected, what can I do?

**Comment by paiineli** [comment]
Hi,

Here are a few suggestions to help you move forward:

Alternative Documentation: Consider providing alternative proof of your student status, such as a student ID, enrollment letter, or course registration document. These can sometimes be accepted.

Contact Support: Reach out to GitHub Support directly and explain your situation. They may be able to provide guidance or alternative options for verification.

Use a Personal Email: If possible, you can try applying with a personal email and include documentation of your student status in your application.

Check University Policies: Sometimes, universities have specific departments that handle student requests. Check if there’s a way to obtain a formal letter or documentation from your university.

I hope this helps, and that you’re able to resolve the issue soon!

**Comment by samus-aran** [comment]
Hi @AylinAghazadeh 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I don't have a school-issued email address, what other official proof of enrollment is accepted?](https://github.com/orgs/community/discussions/111352#i-dont-have-a-school-issued-email-address)
- [What types of evidence does the Student Developer Pack application accept?](https://github.com/orgs/community/discussions/111352#what-types-of-evidence-does-the-student-developer-pack-application-accept)
- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Comment by cesar22416** [comment]
Response #2

**Comment by iamdulanga** [comment]
Hello, @AylinAghazadeh 

1. Actually my campus doesn't provides a student email.
2. But i have tried for 5 times, and 5th time i got accepted.
3. Try the same university name but a little bit different, because if you choose the real one you needs to provide a student email like, the university provides to campus professors and lecturers.
4. Even if you got rejected, try again and again, if your details are actually correct they will accept.

Hope this will help you, if it is @AylinAghazadeh please make my answer as **_answered_**..
Dulanga..
